pagefaultgames/pokerogue
# [Bug] Making FormChangeItems Untransferrable

## What are the changes?

Makes form change items untransferrable.
## Why am I doing these changes?



closes #208 by preventing this scenario.
## What did change?


getTransferrable() really should have been an instance variable instead. The parameter was obselete and was used incorrectly.
### Screenshots/Videos


This G-Max Pikachu is holding a Max Mushroom, but it's no longer giving you the option to transfer it.
![Screenshot (7)](https://github.com/pagefaultgames/pokerogue/assets/63990624/e63bb2eb-bc30-49f5-8388-824c20ee4a44)

## How to test the changes?




Give yourself a form change item. Observe how you can no longer transfer it to other pokemon.
## Checklist
